Dockerfile-debian.template

```diff
@@ -16,7 +16,13 @@
 # limitations under the License.
 ###############################################################################
 
-FROM openjdk:8-jre
+FROM openjdk:8-jre-slim
+
+# Install dependencies
+RUN set -ex; \
+  apt-get update; \
+  apt-get -y install gnupg libsnappy1v5 wget; \
```

Comment:
I guess we could uninstall gnupg and wget as a final step, along with deleting the apt cache.